_layouts/default.html

```diff
@@ -7,12 +7,32 @@
     {% include body-js.html %}
     {% include header.html %}
 
-    <div class="page-content">
+    <div id="page-content" class="page-content">
       <div class="wrapper">
         {{ content }}
       </div>
       {% include footer.html %}
     </div>
 
+    <script>
+        <!-- Execute after the DOM is loaded -->
+        (function() {
+          var content = document.getElementById('page-content');
+          content.addEventListener('scroll', function(e) {
+              var distanceY = window.pageYOffset || content.scrollTop,
+                  shrinkOn = 300,
+                  header = document.querySelector("header.post-header");
```

Comment:
@Eyas Good point about decreasing to a quarter to its size. I don't know how to test it on mobile until its deployed, but it still seems large enough on my desktop.